health/health_test.go

```diff
@@ -175,6 +190,13 @@ func checkEnv(envVar string) string {
 	return value
 }
 
+func getEnv(key string, fallback string) string {
```

Comment:
You can leverage [GetEnv in helpers/env.go](https://github.com/featureform/featureform/blob/main/helpers/env.go#L11-L17)